src/main/java/com/example/bookstoreapp/controller/OrderController.java

```diff
@@ -32,15 +32,16 @@ public class OrderController {
 
     @GetMapping
     @PreAuthorize("hasRole('USER')")
-    @Operation(summary = "Get all orders", description = "Get all orders")
+    @Operation(summary = "Get all orders", description = "Get all orders of current user")
     @ResponseStatus(HttpStatus.OK)
-    public List<OrderDto> getAllOrders() {
-        return orderService.getAllOrders();
+    public List<OrderDto> getAllOrders(Authentication authentication) {
+        return orderService.getAllOrders(getUserId(authentication));
```

Comment:
not sure we need that `getUserId()` at all here, what we can do instead to make it shorter is this:
```suggestion
    public List<OrderDto> getAllOrders(@AuthenticationPrincipal User user) {
        return orderService.getAllOrders(user.getId());
```
same for other similar methods